### DISCUSSION TITLE
Using Pyarmor License - product Name setting

### DISCUSSION THREAD

**Discussion by JL2573** [dashingsoft/pyarmor#1878] [Q&A]
I purchased Pyarmor Pro and would like to do initial setup. 
I am curious about the difference between setting the product name as non-profit and setting it as a unique name.
Is it possible to set a product name and use it for multiple projects?

**Comment by jondy** [comment]
Pyarmor license policy is one product, one license. More information please check http://pyarmor.readthedocs.io/en/latest/licenses.html

**Reply by JL2573** [reply]
What is the difference between setting the product name to "non-profit" and setting it to a specific name when initially setting up pyarmor?

**Comment by jondy** [comment]
"non-profit" means this license is used to obfuscate the scripts which is not sold to get profit

**Reply by JL2573** [reply]
When executing the python -m pyarmor.cli.hwinfo command
Please tell me whether the Machine ID is the number of the actual computer board or an ID created by pyarmor.
